source/generate_tgan_samples.py

```diff
@@ -0,0 +1,12 @@
+import pandas as pd
+import numpy as np
+from tgan.model import TGANModel
+
+model_path = '../demos/Model1'
```

Comment:
The user should be able to choose which model to load, via a command line argument.